app/design/frontend/base/default/template/boltpay/replace.phtml

```diff
@@ -167,6 +167,12 @@ $additionalClasses = $this->boltHelper()->getAdditionalButtonClasses();
 
         replaceCheckout();
 
+        if (typeof Minicart != "undefined") {
+            Minicart.prototype.hideOverlay = function(){
+                onDomReady();
```

Comment:
let's give this better name now that you call this function outside of `DOMContentLoaded` event. `initializeBoltButton`?